vixenNetwork.yml

```
@@ -0,0 +1,25 @@
+---
+name: "Vixen/Blacked/Tushy"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - "https://www.blackedraw.com/"
+      - "https://www.blacked.com/"
+      - "https://www.tushy.com/"
+      - "https://www.tushyraw.com/"
+      - "https://www.vixen.com/"
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@data-test-component="VideoModels"]/a
+    scene:
+      Title: //h1[@data-test-component="VideoTitle"]
+      Performers:
+        Name: $performer
+        URL: $performer/@href
+      Details: //p[@data-test-component="VideoDescription"]
```

```suggestion
      Details: //div[@data-test-component="VideoDescription"]/p
```
This fixes it to pull the details on all the sites.

---

This is the same I posted above for tushy, tushy raw.

---

`//p[@data-test-component="VideoDescription"]|//div[@data-test-component="VideoDescription"]//p`
covers both cases